include/xrpl/protocol/STIssue.h

```diff
@@ -116,19 +144,19 @@ operator!=(STIssue const& lhs, STIssue const& rhs)
 inline bool
 operator<(STIssue const& lhs, STIssue const& rhs)
```

Comment:
could we replace this with `operator<=>` as well ? Also, we should not need `operator!=` anymore